.eslintrc.json

```diff
@@ -3,22 +3,14 @@
     "browser": true,
     "es2021": true
   },
-  "extends": [
-    "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:react-hooks/recommended",
```

Comment:
we need this one `eslint-plugin-react-hooks`
don't forget to install

Reply:
Added